src_cam/mo_neu_wetdep.F90

```diff
@@ -485,17 +519,38 @@ subroutine neu_wetdep_tend(lchnk,ncol,mmr,pmid,pdel,zint,tfld,delt, &
 !if neu wetdep, we have to output it here!
    ! OSLO_AERO begin
    call phys_getopts( history_aerosol_out = history_aerosol)
-   if(history_aerosol)then
-      do m=1,gas_wetdep_cnt
-         wrk_wd(:ncol) = 0.0_r8
-         do k=1,pver
-            !Note sign: tendency is negative, so this becomes a positive flux!
-            wrk_wd(:ncol) = wrk_wd(:ncol)  &
-                 - wd_tend(1:ncol,k,mapping_to_mmr(m))*pdel(:ncol,k)*rgrav !kg/m2/sec
-         end do
-         call outfld('WD_A_'//trim(gas_wetdep_list(m)),wrk_wd(:ncol),ncol,lchnk)
+   do m=1,gas_wetdep_cnt
+      wrk_wd(:ncol) = 0.0_r8
+      do k=1,pver
+        !Note sign: tendency is negative, so this becomes a positive flux!
+        wrk_wd(:ncol) = wrk_wd(:ncol)  &
+          - wd_tend(1:ncol,k,mapping_to_mmr(m))*pdel(:ncol,k)*rgrav !kg/m2/sec
       end do
-   end if
+
+      ! get the index of the gas species that coresponds to the l_spcies system
+      call cnst_get_ind(trim(gas_wetdep_list(m)), l_aero, abort=.false.)
+
+      call outfld('WD_A_'//trim(gas_wetdep_list(m)),wrk_wd(:ncol),ncol,lchnk)
+
+      if ( l_aero == l_so2 ) then
+
+        call outfld('wet_SO2', wrk_wd(:ncol), ncol, lchnk)
+        call outfld('wet_SO2_S', ( wrk_wd(:ncol) * sulfurMassFraction(l_so2) ), ncol, lchnk)
```

Comment:
Where are the `addfld` calls for these? They are usually (i.e., should be) in the same module as the `outfld` calls.